roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -225,8 +225,8 @@
     component: "{{ kibana_component }}"
     logging_component: kibana
     deploy_name: "{{ kibana_name }}"
-    image: "{{ openshift_logging_image_prefix }}logging-kibana:{{ openshift_logging_image_version }}"
-    proxy_image: "{{ openshift_logging_image_prefix }}logging-auth-proxy:{{ openshift_logging_image_version }}"
+    image: "{{ openshift_logging_kibana_image_prefix }}logging-kibana:{{ openshift_logging_kibana_image_version }}"
```

Comment:
I can see a reasoning in the fact, that both images are closely coupled. Frequently only matching versions of kibana and kibana-proxy work together.

That being said, change can easily be implemented. Discussion about the design decisions for the role based image versions happened in https://github.com/openshift/openshift-ansible/pull/3509. This PR tries to implement propagation of the image based versions to appropriate places as previously designed.